.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  push:
+    branches: [main]
+    tags: ["*"]
```

Comment:
This will publish snapshots on every push to `main`, is that what you want?

Reply:
Yes. We should have early feedback on whether the publishing doesn't publish "too much", at the very least.